packages/react/src/generators/component-cypress-spec/component-cypress-spec.ts

```diff
@@ -37,6 +37,8 @@ export function getKnobDefaultValue(property: ts.SyntaxKind): string {
   const resolvedValue = typeNameToDefault[property];
   if (typeof resolvedValue === undefined) {
     return '';
+  } else if (typeof resolvedValue === 'string') {
+    return resolvedValue.replace(' ', '+');
```

Comment:
same as above,not sure replacing just the first blank space is enough

Reply:
Yes, replacing the first one is certainly not enough. I will fix this.